content/quests/events/demevent_rebel_arms_deal.lua

```diff
@@ -43,154 +43,243 @@ QDEF:AddConvo()
+                player:
+                    Ah, calm down. Do you really think they'll be chasing their tails for those "other options" for long?
+                jakes:
+                    Who did you refer them to, exactly?
+                player:
+                    A nobody. They'll be lucky if they could mug the weapon off of this "esteemed weapons dealer".
+                    The Rise'll be eating out of the palm of your hand soon enough.
+                jakes:
+                    That does get the Rise off my back for the now, although I still have these weapons.
+                    I suppose I could call a few favors, get these weapons sold on such short notice.
                 player:
-                    Yeah that might be a problem.
+                    !salute
+                    For that, I wish you luck. 
             ]],
             DIALOG_CONVINCE_CALL_OFF_FAILURE = [[
+                player:
+                    We don't need weapons anymore. You can put them down, because we're entering a democracy now.
                 rise:
-                    [p] No! Revolution needs weapons!
+                    Well, I guess that-
+                jakes:
                     !left
+                    !point
+                    Wait, you guys need to protect Democracy or something.
+                rise:
+                    Y-yeah! We just got Democracy, we need to keep it safe from outside influences!
                 jakes:
+                    That's the spirit.
+                player:
                     !right
-                rise:
-                    And we must all work together to overthrow this corrupt system!
-                    Wouldn't you agree?
                 jakes:
-                    Uh, we are back to square one.
+                    !angry_point
+                    What in Hesh's name are you trying to pull, grifter?
                 player:
-                    !left
+                    !placate
+                    I just saw you were having some business conflicts and I was hoping-
                 jakes:
-                    What are you trying to pull here, grifter?
-                    Trying to put me out of business?
+                    You should hope I don't use any of these weapons on you.
+                    This is still a sale i'd like to make. I don't need grifters butting into other's good business.
             ]],
             OPT_ARREST = "Confront them about dealing with contraband...",
             DIALOG_ARREST = [[
                 jakes:
                     !right
                 player:
                     !left
-                    [p] Say, buddy chum friend pal amigo buddy.
-                    Selling weapons seems very illegal, don't you agree?
+                * You tut into the middle of them with a gait you learned from watching out for admiralty patrols back in your bounty hunting days.
```

Comment:
Only sal is a bounty hunter. The rest is a retired spy and a drunk. And a downtrodden merchant, a spark baron lieutenant, another spark baron kingpin, this random assassin that no one hears about, and also this weird magician, if we are counting mods.
Perhaps use a more general term like "grifting" would work.
Also not sure whether "admiralty" should be capitalized or not.

Reply:
Grifting is probably the best word. Considering the sheer amount of characters mods could provide, I don't think there's a term broad enough to encapsulate literally all of them apart from "the days", which is probably still going to exclude that one person who somehow makes Benjamin button as a character.
As for the Admiralty...yeah that probably needs to be capitalized